package.json

```diff
@@ -22,31 +22,31 @@
     "@fortawesome/free-regular-svg-icons": "6.4.0",
     "@fortawesome/free-solid-svg-icons": "6.4.0",
     "@fortawesome/react-native-fontawesome": "0.2.7",
-    "@hathor/hathor-rpc-handler": "^0.0.2-experimental-alpha",
+    "@hathor/hathor-rpc-handler": "0.0.3-experimental-alpha",
     "@hathor/unleash-client": "0.1.0",
     "@hathor/wallet-lib": "1.10.0",
-    "@json-rpc-tools/utils": "^1.7.6",
+    "@json-rpc-tools/utils": "1.7.6",
     "@notifee/react-native": "5.7.0",
     "@react-native-async-storage/async-storage": "1.19.0",
-    "@react-native-community/netinfo": "11.3.1",
+    "@react-native-community/netinfo": "11.4.1",
     "@react-native-firebase/app": "16.7.0",
     "@react-native-firebase/messaging": "16.7.0",
     "@react-navigation/bottom-tabs": "6.5.8",
     "@react-navigation/native": "6.1.7",
     "@react-navigation/stack": "6.3.17",
-    "@sentry/react-native": "5.6.0",
-    "@walletconnect/core": "^2.15.1",
-    "@walletconnect/react-native-compat": "^2.12.2",
-    "@walletconnect/web3wallet": "^1.14.1",
+    "@reown/walletkit": "1.1.0",
+    "@sentry/react-native": "5.31.0",
+    "@walletconnect/core": "2.17.0",
```

Comment:
Reown crashes on previous version

They didn't move it to the reown namespace yet